### ISSUE TITLE
Map compression/de-compression tool.

### ISSUE THREAD

**Issue by Vaporeon** [SnowyMouse/chimera#83]
Since support for `zstd` compressed maps was removed from invader there should be a standalone tool provided here that can extract or compress these maps.

**Comment by BinToss** [comment]
I agree, somewhat. It would be more convenient if it could simply be thrown it out and never thought about again.
However, because it existed in a stable release, it needs some sort of legacy support even if it is considered deprecated.

On the other hand, a CLI tool doesn't seem like a good fit in the Chimera project's repository.
In other words, it's outside the apparent scope of the project.

If I may make a suggestion, I believe it would be better for it to exist as a standalone project in its own repository. Something along the lines of...
> https://github.com/SnowyMouse/invader-legacy-zstd
The Invader prefix is a placeholder to indicate its relation to other projects.
